src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -77,11 +78,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('repository', help='The repository name for a manifest-only copy of images. Multiple copies supported by passing --repository multiple times.', action='append')
         c.argument('force', help='Overwrite the existing tag of the image to be imported.', action='store_true')
 
-    with self.argument_context('acr config') as c:
+    with self.argument_context('acr config content-trust') as c:
         c.argument('status', help="Indicates whether content-trust is enabled or disabled.", arg_type=get_enum_type(PolicyStatus))
 
     with self.argument_context('acr config retention') as c:
+        c.argument('status', help="Indicates whether retention policy is enabled or disabled.", arg_type=get_enum_type(PolicyStatus))
+        c.argument('registry_name', options_list=['--registry', '-r'])
```

Comment:
Wouldn't this be inconsistent with other `acr config` commands?

Reply:
Is that a problem? We need to reserve `--name` to uniquely identify a retention policy names in the future. Not applicable to `content-trust` though. We can make it `-r` for `content-trust` but that would be a breaking change.

Reply:
It would be good to keep parameter names consistent. I will suggest using a [deprecation](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments).

Reply:
Added deprecation for `-r` and using `-n` instead